subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/Pmd.java

```diff
@@ -159,7 +159,7 @@ public void setPmdClasspath(FileCollection pmdClasspath) {
     }
 
     /**
-     * The built-in rule sets to be used. See the <a href="https://pmd.github.io/pmd-6.20.0/pmd_rules_java.html">official list</a> of built-in rule sets.
+     * The built-in rule sets to be used. See the <a href="https://pmd.github.io/pmd-6.21.0/pmd_rules_java.html">official list</a> of built-in rule sets.
```

Comment:
Is there a stable link for these?

Reply:
I think we can use `latest` but I'm not sure we should. Given the rules available depend on the tool version (and our default), these urls should match the default version. WDYT?

Version-independent url: https://pmd.github.io/latest/pmd_rules_java.html

Reply:
Yeah, I suppose you're right there.